trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9] if @nameable.correct_name.length >= 10
```

Comment:
- [ ] What if `correct_name.length` is < 10 , We have to handle that use case also.
- Eg:- `@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`

Reply:
I really appreciate your suggestion!